### ISSUE TITLE
(Trac #165) eventual.py has two unnecessary lines

### ISSUE THREAD

**Issue by foolscap-trac** [foolscap-trac/test-foolscap-import-5#165]
|  | trac:ivank reported |
|-|-|
| Trac ID | trac#165 |
| Type | defect |
| Created | 2010-11-10 09:19:21Z |
| Version | 0.5.1 |
| Resolution | fixed |

In eventual.py, the body of this `if` block is never run (unless of course some callback mutates `_events` instead of calling `append`)

```
        if self._events and not self._timer:
            self._timer = reactor.callLater(0, self._turn)
```

I don't think it's needed because `_SimpleCallQueue.append` sets `self._timer`.

**Comment by foolscap-trac** [comment]
| <img alt="warner's avatar" src="https://avatars.githubusercontent.com/u/27146?v=4" width="50" height="50"> | trac:warner commented |
|-|-|

I concur. Fixed in [7904b98a](https://github.com/warner/foolscap/commit/7904b98a03324479b6c26a0b7a8ba51052302a09). Thanks!

**Comment by foolscap-trac** [comment]
| <img alt="warner's avatar" src="https://avatars.githubusercontent.com/u/27146?v=4" width="50" height="50"> | trac:warner set resolution to `fixed` |
|-|-|
